src/js/tabs/pid_tuning.js

```diff
@@ -12,6 +12,7 @@ TABS.pid_tuning = {
     SETPOINT_WEIGHT_RANGE_LEGACY: 2.54,
     activeSubtab: 'pid',
     analyticsChanges: {},
+    THR_CHAN_INDEX: 3,
```

Comment:
This can't be defined, what if I use TAER1234 channel map?

Reply:
Good point! I just looked at how the rate curve is drawn and there the values are simply referenced by index, like [here](https://github.com/betaflight/betaflight-configurator/blob/master/src/js/tabs/pid_tuning.js#L1716). And then in `fc.js` the array is defined [like this](https://github.com/betaflight/betaflight-configurator/blob/master/src/js/fc.js#L158). So this gave me the impression that the values are normalized by the firmware before they are sent over MSP.

Reply:
It seems to be correct. I interpreted it wrong because of how msp rx works. 😕 This works as expected, my bad.

Reply:
Hm, so it seems the rate curve is also bugged, isn't it? It should use `RC_MAP` to determine the indices, if I'm not mistaken.